libnetwork/networkdb/networkdb_test.go

```diff
@@ -241,7 +241,7 @@ func TestNetworkDBJoinLeaveNetworks(t *testing.T) {
 	closeNetworkDBInstances(t, dbs)
 }
 
-func TestNetworkDBCRUDTableEntry(t *testing.T) {
+func TestFlakyNetworkDBCRUDTableEntry(t *testing.T) {
```

Comment:
Why are you renaming these tests? The infrastructure to rerun flaky tests hasn't been backported to the 23.0 branch. Do we need it?